From fb31b46c0e196a59f1562c827d23eac4eada7fdf Mon Sep 17 00:00:00 2001
From: Paul Duffin <paulduffin@google.com>
Date: Mon, 6 Mar 2017 16:18:15 +0000
Subject: [PATCH 141/172] Removed littlemock dependency and cleanup

The tests_camera classes are legacy tests that have not been
built for nearly 4 years. This removes those classes and removes
the littlemock and dexmaker dependencies that are no longer
needed.

Bug: 32912773
Test: build, install and run using am instrument
Change-Id: I623d52fd5471f75622c2175061467ad5519dad13
---
 tests/Android.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/Android.mk b/tests/Android.mk
index 920f8f0ff..cd768e06b 100644
--- a/tests/Android.mk
+++ b/tests/Android.mk
@@ -6,8 +6,6 @@ LOCAL_MODULE_TAGS := tests
 
 LOCAL_SDK_VERSION := 16
 
-LOCAL_STATIC_JAVA_LIBRARIES := littlemock dexmaker
-
 # Include all test java files.
 LOCAL_SRC_FILES := $(call all-java-files-under, src)
 
-- 
2.17.1

